sdk/storage/storage-blob-changefeed/src/AvroReaderFactory.ts

```diff
@@ -0,0 +1,23 @@
+import { AvroReadable, AvroReader } from '../../storage-internal-avro/src';
+
+export class AvroReaderFactory {
+  public buildAvroReader(dataStream: AvroReadable): AvroReader;
+  public buildAvroReader(
+    dataStream: AvroReadable,
+    headerStream: AvroReadable,
+    blockOffset: number,
+    eventIndex: number): AvroReader;
+
+  public buildAvroReader(
+    dataStream: AvroReadable,
```

Comment:
it look like if headerStream is not provided, dataStream start from 0, i.e. it's also headerStream?

Reply:
Kind of. Do you have any good advice on the naming?
I am thinking overloading with
```
public buildAvroReader(
    wholeStream: AvroReadable,
  );
public buildAvroReader(
    dataStream: AvroReadable,
    headerStream: AvroReadable,
    blockOffset: number,
    eventIndex: number
  );
```